rails_docker/Dockerfile

```diff
@@ -12,58 +12,63 @@ ENV RACK_ENV=$RUBY_ENV \
     BUILD_ENV=$BUILD_ENV \
     ASSET_HOST=$ASSET_HOST
 
-ENV APP_HOME=/#{app_name} \
-    PORT=80
+ENV APP_HOME=/usr/src/#{app_name}
```

Comment:
Any benefits to move to this subdirectory?

Reply:
Just a convention. Would you prefer to have it in the root?